src/shared/components/CopyToClipboard.tsx

```diff
@@ -0,0 +1,48 @@
+// Libraries
+import React, {FC, ChangeEvent} from 'react'
+
+const copy = async (text: string): Promise<boolean> => {
+  let result: boolean
+  try {
+    if (navigator.clipboard) {
+      // All browsers except IE
+      await navigator.clipboard.writeText(text)
+    } else {
+      // IE
+      const successful = document.execCommand('copy')
+      if (!successful) {
+        throw new Error('Copy command was unsuccessful using execCommand')
+      }
```

Comment:
@asalem1 `document.execCommand` is a polyfill for the clipboard API in IE

Reply:
Gotcha. I misunderstood what you had mentioned in yesterday's stand-up about IE support. Ya i looked at the docs and it said it was deprecated and might be discontinued at any moment. I'm wondering if there's anything else we can do rather than this or if this is our only option

Reply:
I think this is our only option for supporting IE to copy to clipboard. 

This is the main browser API that `copy-to-clipboard` library used in their code for copying text to clipboard. `copy-to-clipboard` is a dependency of `react-copy-to-clipboard` that we want to remove in this PR. Since we use the recommended method to copy text to clipboard for other browsers except IE, i.e. `navigator.clipboard.writeText`, we should be fine here. 

Gary confirmed with the product team that we do not support InfluxData Cloud for IE, so I think we can either remove this part of the code for IE or keep it here. What do you think? @asalem1

Reply:
Sounds good. Let's do it :)